core/src/main/java/org/elasticsearch/bootstrap/Bootstrap.java

```diff
@@ -93,12 +93,14 @@ public static void initializeNatives(Path tmpFile, boolean mlockAll, boolean sec
                 throw new RuntimeException("don't run elasticsearch as root.");
             }
         }
-        
+
         // enable secure computing mode
         if (seccomp) {
             Natives.trySeccomp(tmpFile);
+        } else {
+            logger.warn("Not enabling seccomp because it was explicitly disabled");
         }
```

Comment:
Well, i dont really think it is. I guess i don't understand why we are running backwards tests this way, versus doing real integration tests with the rest suite and similar (like we do in qa). All the stuff needed for that is published as maven artifacts. One of these approaches is a "fantasy world" that isn't quite realistic, it requires manual setup and is difficult to run, and flaky as far as jenkins goes. the other is automatic, realistic and reliable... what am I missing?